### ISSUE TITLE
[page structure, example]  aria-label missing?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-tutorials#736]
_from [e-mail](https://lists.w3.org/Archives/Public/wai-site-comments/2023Nov/0004.html) :_

below [approach 2: Using aria-label on page](https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-label) states "View a complete code example with regions and labels."

however on ["complete code example" page](https://www.w3.org/WAI/tutorials/page-structure/example/), there is no aria-label. only aria-labelledby are given.

**Comment by brianelton** [comment]
I moved the link to the using aria-labelledby section.
